packages/ui/src/Banner.tsx

```diff
@@ -77,18 +77,21 @@ export const Banner = ({
     if (type === ACTION) {
       return (
         <Box alignItems="center" display="block" justifyContent="center" width={40}>
+          {/* accessibility-hint handled in IconButton component */}
+          {/* eslint-disable-next-line react-native-a11y/has-accessibility-hint */}
           <IconButton
-            accessibilityLabel=""
-            icon="arrow-right"
-            prefix="fas"
+            accessibilityLabel="arrow-right"
+            iconName="arrow-right"
             onClick={(): void => onClick?.()}
           />
         </Box>
       );
     } else if (type === DISMISS) {
       return (
         <Box alignItems="center" display="block" justifyContent="center" width={40}>
-          <IconButton accessibilityLabel="" icon="circle-xmark" prefix="fas" onClick={dismiss} />
+          {/* accessibility-hint handled in IconButton component */}
+          {/* eslint-disable-next-line react-native-a11y/has-accessibility-hint */}
+          <IconButton accessibilityLabel="" iconName="circle-xmark" onClick={dismiss} />
```

Comment:
Not your fault obviously but it's annoying we can get away with putting in an empty string like this - thank you for updating so many examples to avoid this!!!!

Reply:
I tried to leave this sill minimal changes since this component still needs to be updated. I think once it is actually updated it should include a correct label via props

Reply:
Yes 100 percent agree!